# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jsaw/epplets-extra16/e-dial.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- E-Dial/COMPILE.orig-rock-maint	2003-08-29 08:39:04.000000000 +0200
+++ E-Dial/COMPILE	2003-08-29 09:03:36.000000000 +0200
@@ -2,7 +2,7 @@
 # Run this to install E-Dial on your system.
 
 NAME=E-Dial
-cc edial.c epplet.c -o $NAME.epplet -O2 `imlib-config --libs --cflags`
+$CC $CFLAGS edial.c epplet.c -o $NAME.epplet -O2 `imlib-config --libs --cflags` -L$E16/lib -lepplet
 cp -f $NAME.epplet $EBIN/$NAME.epplet
 chmod a+x $EBIN/$NAME.epplet
 
